app/Nova/TravelAssignment.php

```diff
@@ -93,6 +101,36 @@ public function fields(NovaRequest $request): array
                             ?->id
                 ),
 
+            Code::make('Matrix Itinerary')
+                ->json()
+                ->rules('nullable', 'json', new MatrixItineraryDataStructure())
+                ->required()
+                ->help(
+                    'If this trip includes airfare, you must provide an itinerary in Matrix JSON format. Search '.
```

Comment:
It would be slick to add the `required` rule dynamically if the `Travel` has airfare enabled. Not a deal breaker.

Reply:
Yep, I'll handle that when I redo TA creation.